bgpd/bgp_filter.h

```diff
@@ -34,6 +39,7 @@ struct as_list {
 
 	struct as_filter *head;
 	struct as_filter *tail;
+	struct aspath_exclude_list *depend_of_list;
```

Comment:
Why not just `exclude_list`? `depend_of_list` looks to generic IMO...

Reply:
done
but  depend_of_list explain, the role of that list.
exclude list exhibit the  type of that list
now, nothing is showing the reason of the presence of that list!